src/app/clusters/bindings/BindingManager.cpp

```diff
@@ -160,7 +161,16 @@ void BindingManager::HandleDeviceConnected(OperationalDeviceProxy * device)
         {
             fabricToRemove = entry.fabricIndex;
             nodeToRemove   = entry.nodeId;
-            mBoundDeviceChangedHandler(entry, device, pendingNotification.mContext);
+
+            BindingManagerContext * context = static_cast<BindingManagerContext *>(pendingNotification.mContext);
```

Comment:
Given we now always store a BindingManagerContext in the pending notifications, can't we change the type of `mContext` to `BindingManagerContext *`?

Reply:
Done.